src/nvim/getchar.c

```diff
@@ -1730,14 +1722,17 @@ static int vgetorpeek(const int advance)
     if (typebuf.tb_no_abbr_cnt == 0)
       typebuf.tb_no_abbr_cnt = 1;             /* no abbreviations now */
   } else {
+    int timedout = FALSE;                     /* waited for more than 1 second
```

Comment:
Can you have a single PR dealing with the scope reduction changes?

Mixing scope reductions and the changes you're doing here makes it hard to review and understand that your changes preserve the current behavior of the code.